definitions/infra-win_service/summary_metrics.yml

```diff
@@ -1,15 +1,21 @@
 serviceName:
-  unit: STRING
-  tag:
-    key: name
   title: Service Name
-hostName:
   unit: STRING
-  tag:
-    key: host.hostname
+  query:
+    select: latest(name)
+    from: Metric
+    eventId: entity.guid
+hostName:
   title: Host Name
-startMode:
   unit: STRING
-  tag:
-    key: windows_service_start_mode
+  query:
+    select: latest(host.hostname)
+    from: Metric
+    eventId: entity.guid
+startMode:
   title: Start Mode
+  unit: STRING
```

Comment:
The `STRING` unit is not actually supported for NRDB queries ([docs](https://github.com/newrelic/entity-definitions/blob/main/docs/summary_metrics.md#metric-unit)), our advice is to use tags instead, could you let us know why that approach wasn't working? thanks!